src/plugins/plot/inspector/PlotOptionsItem.vue

```diff
@@ -132,7 +132,8 @@ export default {
   },
   data() {
     return {
-      expanded: false
+      expanded: false,
+      status: null
```

Comment:
we reference `status` as part of the render, causing vue to puke a warning

Reply:
nice catch